docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -342,7 +342,7 @@ spec:
       swiftRing:
         ringReplicas: 1
       swiftStorage:
-        replicas: 0
+        replicas: 1
```

Comment:
This needs to be 0. The deployment still uses the storage nodes on the old deployment at this stage.